simple_admittance_controller:
    type: simple_admittance_controller/SimpleAdmittanceController
    m_tr: 5
    m_rot: 0.1
    d_tr: 10
    d_rot: 0.4
    only_transl: false
    only_x: false
    FT_filter_level: 0
    lowpass_filter_cutoff_freq: 1000
    eth_interface_name: enxa0cec88b0292
